manifests/install.yaml

```diff
@@ -360,6 +368,10 @@ spec:
                           name:
                             type: string
                         type: object
+                      ref:
+                        description: Ref is reference to another source within sources
```

Comment:
Maybe we could add that this field is "useless" when used if a "source" instead of "sources". Wdyt?